src/server/routes/register.ts

```diff
@@ -337,153 +353,179 @@ export const OktaRegistration = async (
 	const registrationLocation: RegistrationLocation | undefined =
 		getRegistrationLocation(req);
 
-	// OKTA IDX API FLOW
-	// Attempt to register the user with Okta using the IDX API
-	// and specifically using passcodes.
-	// If there are specific failures, we fall back to the legacy Okta registration flow
-	if (passcodesEnabled && !useOktaClassic) {
-		try {
-			const introspectResponse = await startIdxFlow({
-				req,
-				res,
-				authorizationCodeFlowOptions: {
-					confirmationPagePath: '/welcome/review',
-				},
-				consents,
-				request_id,
-			});
+	try {
+		const introspectResponse = await startIdxFlow({
+			req,
+			res,
+			authorizationCodeFlowOptions: {
+				confirmationPagePath: '/welcome/review',
+			},
+			consents,
+			request_id,
+		});
 
-			// check if we have the `select-enroll-profile` remediation property which means registration is allowed
-			const introspectEnrollProfileCheck =
-				introspectResponse.remediation.value.find(
-					({ name }) => name === 'select-enroll-profile',
-				);
+		// check if we have the `select-enroll-profile` remediation property which means registration is allowed
+		const introspectEnrollProfileCheck =
+			introspectResponse.remediation.value.find(
+				({ name }) => name === 'select-enroll-profile',
+			);
```

Comment:
Are we able to use `validateRemediation` here?

Reply:
Ah it's something I missed! Have added an `validateEnrollRemediation` method.

There was also a place above in the code where I added `validateIntrospectRemediation` too!